config/boards/orangepi5-plus.wip

```diff
@@ -0,0 +1,34 @@
+# Rockchip RK3588 octa core 4/8/16GB RAM SoC NVMe USB3 USB-C 2.5GbE eMMC
+BOARD_NAME="Orange Pi 5 plus"
+BOARDFAMILY="rockchip-rk3588"
+BOOTCONFIG="orangepi_5_plus_defconfig" # vendor name, not standard, see hook below, set BOOT_SOC below to compensate
+BOOT_SOC="rk3588"
+KERNEL_TARGET="legacy"
+FULL_DESKTOP="yes"
+BOOT_LOGO="desktop"
+BOOT_FDT_FILE="rockchip/rk3588-orangepi-5-plus.dtb"
+BOOT_SCENARIO="spl-blobs"
+BOOT_SUPPORT_SPI="yes"
+BOOT_SPI_RKSPI_LOADER="yes"
+IMAGE_PARTITION_TABLE="gpt"
+SKIP_BOOTSPLASH="yes" # Skip boot splash patch, conflicts with CONFIG_VT=yes
+BOOTFS_TYPE="ext4"
+
+
+function post_family_tweaks__orangepi5plus_naming_audios() {
+	display_alert "$BOARD" "Renaming orangepi5 audios" "info"
+
+	mkdir -p $SDCARD/etc/udev/rules.d/
+	echo 'SUBSYSTEM=="sound", ENV{ID_PATH}=="platform-hdmi0-sound", ENV{SOUND_DESCRIPTION}="HDMI0 Audio"' > $SDCARD/etc/udev/rules.d/90-naming-audios.rules
```

Comment:
Can you also add HDMI1

Reply:
hdmi1-sound is disabled in dtb.

Reply:
> hdmi1-sound is disabled in dtb.

Nope it's enabled https://github.com/armbian/linux-rockchip/blob/rk-5.10-rkr4/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dtsi#L377

Reply:
okay, i had add hdmi1 sound rule.